### PR TITLE
Update Authentication.md

### PR DESCRIPTION
**Pull request by DoviMaj**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Removed repeated words, added commas, and corrections.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

**Comment by DoviMaj** [comment]
sorry for making a new PR, I realized those errors after submitting the previous one

File: nodeJS/authentication/Authentication.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ By the end of this lesson, you should be able to do the following:
...
 
 ~~~
 npm install express express-session mongoose passport passport-local ejs
 ~~~
 
-Next let's create our `app.js`:
+Next, let's create our `app.js`:
 
-**IMPORTANT NOTE**: For the moment we are saving our users with just a plain text password.  This is a _really_ bad idea for any real-world project. At the end of this lesson you will learn how to properly secure these passwords using bcrypt. Don't skip that part.
+**IMPORTANT NOTE**: For the moment we are saving our users with just a plain text password.  This is a _really_ bad idea for any real-world project. At the end of this lesson, you will learn how to properly secure these passwords using bcrypt. Don't skip that part.
```
Comment (twalton83) [inline nodeJS/authentication/Authentication.md:36]: I think this comma is unnecessary.

File: nodeJS/authentication/Authentication.md [hunk 2]
```diff
@@ -257,7 +257,7 @@ and then edit your view to make use of that object like this:
 </html>
 ~~~
 
-So, this code checks to see if there is a user defined... if so it offers a welcome message, and if NOT then it shows the login form.  Neat!
+So, this code checks to see if there is a user-defined... if so it offers a welcome message, and if NOT then it shows the login form.  Neat!
```
Comment (twalton83) [inline nodeJS/authentication/Authentication.md:260]: This hyphen isn't grammatically correct here.

File: nodeJS/authentication/Authentication.md [hunk 3]
```diff
@@ -289,7 +289,7 @@ If you insert this code somewhere between where you instantiate the passport mid
 
 Now, let's go back and learn how to securely store user passwords so that if anything ever goes wrong, or if someone gains access to our database, our user passwords will be safe.  This is _insanely_ important, even for the simplest apps, but luckily it's also really simple to set up.
 
-First `npm install bcryptjs`. There is another module called `bcrypt` that does the same thing, but it is written in C++ and is sometimes a pain to get installed. The C++ `bcrypt` is technically faster, so in the future it might be worth getting it running, but for now the modules work the same so we can just use `bcryptjs`.
+First `npm install bcryptjs`. There is another module called `bcrypt` that does the same thing, but it is written in C++ and is sometimes a pain to get installed. The C++ `bcrypt` is technically faster, so in the future, it might be worth getting it running, but for now, the modules work the same so we can just use `bcryptjs`.
```
Comment (twalton83) [inline nodeJS/authentication/Authentication.md:292]: I'm not sure this extra comma is necessary

Comment (DoviMaj) [reply nodeJS/authentication/Authentication.md:292]: but for now or so in the future?

Comment (DoviMaj) [reply nodeJS/authentication/Authentication.md:292]: Grammarly told me so but its not 100% accurate

File: nodeJS/authentication/Authentication.md (hunk 1 above)
Comment (marvingay) [reply nodeJS/authentication/Authentication.md]: The comma would be grammatically correct.

Comment (DoviMaj) [reply nodeJS/authentication/Authentication.md]: il add it back

File: nodeJS/authentication/Authentication.md (hunk 3 above)
Comment (DoviMaj) [reply nodeJS/authentication/Authentication.md:292]: let me know...

Comment (twalton83) [reply nodeJS/authentication/Authentication.md:292]: Sorry, "so in the future". It just feels like that's unnecessarily breaking up that piece of the sentence.

Comment (DoviMaj) [reply nodeJS/authentication/Authentication.md]: Ok. changed that.

**Review by twalton83 — APPROVED** [review]
Thank you for this contribution.

**Pull request merged**